clone.js

```diff
@@ -0,0 +1,104 @@
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+const path = require('path');
+const fs = require('fs')
+
+readline.question(`Wat is de naam van de nieuwe webcomponent? `, (naam) => {
```

Comment:
Spijtig dat readline.question geen promise teruggeeft zodat we await zouden kunnen doen. Is wat ambetant dat je vragen moet nesten. Maar het is niet echt de moeite om zelf iets aan te doen.